model_analyzer/config/generate/search_config.py

```diff
@@ -18,13 +18,43 @@ class SearchConfig:
     Defines all dimensions to search
     """
 
-    def __init__(self, dimensions):
+    def __init__(self,
+                 dimensions,
+                 neighborhood_radius=2,
```

Comment:
This class gets passed to Neighborhood, but the radius value here should explicitly NOT be used. I'm thinking I need to create another config for Neighborhood which is a subset of this class to make sure it is never accidentally used